packages/scene-composer/src/components/panels/SceneNodeInspectorPanel.tsx

```diff
@@ -57,7 +57,7 @@ export const SceneNodeInspectorPanel: React.FC = () => {
       description: 'Expandable Section title',
     },
     [KnownComponentType.DataBinding]: {
```

Comment:
we should rename `KnownComponentType.DataBinding` to `KnownComponentType.EntityBinding` to not get confused internally